roles/openshift_aws/defaults/main.yml

```diff
@@ -66,15 +106,15 @@ openshift_aws_elb_dict:
         healthy_threshold: 2
       idle_timeout: 400
       listeners:
-      - protocol: tcp
-        load_balancer_port: 80
-        instance_protocol: ssl
-        instance_port: "{{ openshift_master_api_port }}"
-      - protocol: ssl
-        load_balancer_port: "{{ openshift_master_api_port }}"
-        instance_protocol: ssl
-        instance_port: "{{ openshift_master_api_port }}"
-        ssl_certificate_id: "{{ openshift_aws_elb_cert_arn }}"
+        - protocol: tcp
```

Comment:
We default to using zero padding for lists.  yaml doesn't care either way, except you need to be consistent on a per-file basis.

Please restore the indents and make your additions conform so we don't have all these extra change lines filling up the PR.